README.md

```diff
@@ -1,2 +1,69 @@
 # jupyter-packaging
+
 Tools to help build and install Jupyter Python packages
+
+## Install
+
+`pip install jupyter-packaging`
+
+## Usage
+
+Below is an example `setup.py` that uses jupyter-packaging:
+
+```py
```

Comment:
@blink1073 This is intended to be a bare bones example of a `setup.py` using jupyter-packaging. Let me know if anything looks off...